Makefile

```diff
@@ -151,6 +151,7 @@ docker-push:  ## Push the docker image
 
 bundle: operator-sdk manifests  ## Generate bundle manifests and metadata, then validate generated files.
 	ls -d config/crd/bases/* | grep -v metallb.io_metallbs | xargs -I{} cp {} bundle/manifests/
+	rm bundle/manifests/metallb.io_servicel2statuses.yaml # TODO remove when metallb support is added
```

Comment:
this file has been deleted, so is no op command, right?

Reply:
this is upstream and just ported on the openshift version, where we mirror what we have u/s.
In any case, this is required because the alignement scripts that pick the changes in metallb would copy that again.